ChromiumHtmlToPdfLib/Browser.cs

```diff
@@ -254,7 +254,7 @@ private enum PageLoadingState
 
                 var pageSetDocumentContent = new Message { Method = "Page.setDocumentContent" };
                 pageSetDocumentContent.AddParameter("frameId", frameResult.Result.FrameTree.Frame.Id);
-                pageSetDocumentContent.AddParameter("html", html);
+                pageSetDocumentContent.AddParameter("html", html!);
```

Comment:
suppressions like this used is several places as `IsNullOrWhiteSpace` nullability effects not tracked by target frameworks